x/signal/legacy_test.go

```diff
@@ -86,119 +64,102 @@ func (s *LegacyUpgradeTestSuite) SetupSuite() {
-		Plan: types.Plan{
-			Name:   "v1",
-			Height: 20,
-			Info:   "rough social consensus",
-		},
-	}
-	dep := sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000000000)))
-	acc := s.unusedAccount()
-	accAddr := getAddress(acc, s.cctx.Keyring)
-	msg, err := v1.NewMsgSubmitProposal([]sdk.Msg{&sss}, dep, accAddr.String(), "")
-	require.NoError(t, err)
-
-	// submit the transaction and wait a block for it to be included
-	txClient, err := testnode.NewTxClientFromContext(s.cctx)
-	require.NoError(t, err)
-	subCtx, cancel := context.WithTimeout(s.cctx.GoContext(), time.Minute)
-	defer cancel()
-	_, err = txClient.SubmitTx(subCtx, []sdk.Msg{msg}, blobfactory.DefaultTxOpts()...)
-	// As the type is not registered, the message will fail with unable to resolve type URL
-	require.Error(t, err)
-	code := err.(*user.BroadcastTxError).Code
-	require.EqualValues(t, 2, code, err.Error())
-}
-
-func (s *LegacyUpgradeTestSuite) TestIBCUpgradeFailure() {
-	t := s.T()
-	plan := types.Plan{
-		Name:   "v2",
-		Height: 20,
-		Info:   "this should not pass",
-	}
-	upgradedClientState := &ibctmtypes.ClientState{}
-
-	upgradeMsg, err := ibctypes.NewUpgradeProposal("Upgrade to v2!", "Upgrade to v2!", plan, upgradedClientState)
-	require.NoError(t, err)
-
-	dep := sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000000000)))
-	acc := s.unusedAccount()
-	accAddr := getAddress(acc, s.cctx.Keyring)
-	msg, err := v1beta1.NewMsgSubmitProposal(upgradeMsg, dep, accAddr)
-	require.NoError(t, err)
-
-	// submit the transaction and wait a block for it to be included
-	txClient, err := testnode.NewTxClientFromContext(s.cctx)
-	require.NoError(t, err)
-	subCtx, cancel := context.WithTimeout(s.cctx.GoContext(), time.Minute)
-	defer cancel()
-	_, err = txClient.SubmitTx(subCtx, []sdk.Msg{msg}, blobfactory.DefaultTxOpts()...)
-	require.Error(t, err)
-	code := err.(*user.ExecutionError).Code
-	require.EqualValues(t, 9, code) // we're only submitting the tx, so we expect everything to work
-	assert.Contains(t, err.Error(), "ibc upgrade proposal not supported")
-}
-
-func getAddress(account string, kr keyring.Keyring) sdk.AccAddress {
-	rec, err := kr.Key(account)
-	if err != nil {
-		panic(err)
-	}
-	addr, err := rec.GetAddress()
-	if err != nil {
-		panic(err)
-	}
-	return addr
-}
+// func getAddress(account string, kr keyring.Keyring) sdk.AccAddress {
+// 	rec, err := kr.Key(account)
+// 	if err != nil {
+// 		panic(err)
+// 	}
+// 	addr, err := rec.GetAddress()
+// 	if err != nil {
+// 		panic(err)
+// 	}
+// 	return addr
+// }
+
+// func (s *LegacyUpgradeTestSuite) unusedAccount() string {
+// 	s.mut.Lock()
+// 	acc := s.accounts[s.accountCounter]
+// 	s.accountCounter++
+// 	s.mut.Unlock()
+// 	return acc
+// }
+
+// TODO: Finish test refactor. The x/circuit AnteHandler does not filter a MsgSubmitProposal's msgs.
+// It is blocked by x/circuit after voting on proposal when its msgs are executed.
+// func (s *LegacyUpgradeTestSuite) TestNewGovUpgradeFailure() {
```

Comment:
any thoughts on these tests? 

they either need to be completely refactored or removed, see comment

Reply:
we can do in a follow up